class-components/src/components/Floyt/tests/Flyout.test.tsx

```diff
@@ -0,0 +1,110 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import configureStore, { MockStoreEnhanced } from 'redux-mock-store';
+import Flyout from '../Flyout';
+import { unselectAll } from '../../../store/searchSlice';
+
+interface People {
+  name: string;
+  description?: string;
+  url: string;
+  height: string;
+  eye_color: string;
+  skin_color: string;
+  mass: string;
+  hair_color: string;
+  birth_year: string;
+  gender: string;
+}
+
+interface SearchState {
+  searchTerm: string;
+  selectedItemId: string | null;
+  selectedItem: People | null;
+  items: People[];
+  selectedItems: string[];
+}
```

Comment:
It's okay to import these from the relevant folder and not to re-declare them in tests